src/plugin/jquery-plugin.js

```diff
@@ -256,9 +256,9 @@
      *   Describes layers added to the map
      * @property {boolean} [autoresize=true]
      *   Resize the map on <code>window.resize</code> (initialization only)
-     * @property {string} [tileServer]
+     * @property {string} [tileUrl]
      *   The open street map tile server spec default:
-     *   <code>http://tile.openstreetmap.org/<zoom>/<x>/<y>.png</code>
+     *   <code>http://tile.openstreetmap.org/&lt;zoom>/&lt;x>/&lt;y>.png</code>
```

Comment:
I'm not sure about exact rules for html escape, but this seems to work.

![screen shot 2015-05-08 at 11 40 58 am](https://cloud.githubusercontent.com/assets/31890/7539762/277552a4-f577-11e4-8eaf-36b8e55300a2.png)